monitoring/kafka/alerts.test.yaml

```
@@ -35,6 +35,7 @@ tests:
               description: 'Kafka: Broker count is down'
             exp_labels:
               severity: warning
+              service: ${service}
```

in the dashboard, we use `Cluster` as the label to identify specific instances → would be more appropriate than `service` (which is an implementation detail in k8s context)

```suggestion
              cluster: ${service}
```